Src/Base/AMReX_FabArray.H

```diff
@@ -864,6 +938,31 @@ public:
     Vector<char*>       fb_send_data;
     Vector<MPI_Request> fb_send_reqs;
     int                 fb_tag;
+
+    //! Data used in non-blocking ParallelCopy
+    const CPC*           pc_cpc = nullptr;
+    const FabArray<FAB>* pc_src = nullptr;
+    CpOp                 pc_op;
+    int                  pc_tag;
+    int                  pc_actual_n_rcvs;
+    IntVect              pc_snghost, pc_dnghost;
+    Periodicity          pc_period;
+    int                  pc_SC, pc_NC, pc_DC;
+
+    //
+    char*                pc_the_recv_data = nullptr;
+    char*                pc_the_send_data = nullptr;
+    Vector<int>          pc_recv_from;
+    Vector<char*>        pc_recv_data;
+    Vector<std::size_t>  pc_recv_size;
+    Vector<MPI_Request>  pc_recv_reqs;
+//    Vector<MPI_Status>  pc_recv_stat;  // why kept in FB? Needed in PC?
```

Comment:
I think it's because we have `FillBoundary_test`, we must save the MPI_Status for FillBoundary.  Not needed in PC.